site/aboutlinda.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE HTML>
+
+<html>
+
+<head>
+	<title>Linda Diep - About me</title>
+	<link rel="stylesheet" href="style.css" >
+</head>
+
+<body>
+	
+	<div id="header">
+  		<a href="aboutlinda.html">About Me</a> | <a href="portfolio.html" style="text-decoration:none">Portfolio</a><br>
+  		<h1>Linda Diep</h1>
+	</div>
+
+	<div div="section">
+		<img src="images/cherryblossoms.gif" alt="Washington DC" title="Washington DC" align="right">
+
+		I am a communications professional based in Washington, DC looking for creative and fun projects to work on.
+	</div>
+
+	<div id="footer">
+		<center>
+		<table>
+ 			<tr>
+			<td><a href="https://www.linkedin.com/in/linda-diep-87624923" target="_blank">LinkedIn</a></td> 
+			<td><a href="http://www.twitter.com/lindaack" target="_blank">Twitter</a></td>
+			<td><a href="mailto:lxdiep07@gmail.com">Email</a></td>
+ 		</tr>
```

Comment:
be mindful of your indentation here with the tr tag. One thing I like to do is run my code through a formatter to make sure things are aligned correctly, but be sure to still doublecheck to make sure it's formatted how you want it. http://codebeautify.org/htmlviewer/